config/samples/translator.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: ibmcloud.ibm.com/v1alpha1
 kind: Service
 metadata:
   name: mytranslator
-  namespace: default
+  namespace: name1
   annotations:
```

Comment:
can we get rid of `namespace` in this sample, so it can be used in any namespace ?